Library/Homebrew/os/mac/xcode.rb

```diff
@@ -234,6 +234,30 @@ def detect_version_from_clang_version
       def default_prefix?
         prefix.to_s == "/Applications/Xcode.app/Contents/Developer"
       end
+
+      sig { params(swift_version: String).returns(String) }
+      def minimum_version_for_swift(swift_version)
```

Comment:
The whole versioning thing feels like a relic from when homebrew did source builds. Wouldn't it be much easier if we just said "latest xcode" if you want swift.

Reply:
"latest xcode" means different things for different macOS versions, so it isn't always sufficient for a formula. That's why we specify versions in some Xcode dependencies rather than just `depends_on :xcode`  to mean "latest xcode".

Reply:
Isn't that what we have DSLs for to specify required macOS version? It's weird to me that xcode (and soon swift) wouldn't have to follow Homebrew's evergreen policy.

Reply:
We've never had an evergreen policy on macOS development tools, and won't as long as `MacOS::Xcode.minimum_version` and `MacOS::CLT.minumum_version` exists.

Reply:
That's why I think it is weird and the whole thing seems opposed to how the rest of Homebrew functions. If anything I would expect a swift dependency to use the last version available, just as all other formula do.